pkg/collector/corechecks/cluster/orchestrator/transformers/k8s/pod.go

```diff
@@ -262,6 +262,19 @@ func convertResourceRequirements(rq corev1.ResourceRequirements, containerName s
 		}
 	}
 
+	// Fill non-default values (other than CPU and Memory)
+	for resourceName, quantity := range rq.Limits {
+		if _, found := limits[resourceName.String()]; !found {
```

Comment:
nit: you can remove the existence check here, the same resource can't be defined twice at container level

Reply:
I wrote this existence check so that I don't override the existing value for CPU and Memory that may have different unit (milli/value) because of the code above mine